.gitignore

```diff
@@ -46,9 +46,14 @@ python-sdk/docs/autoapi/
 **/dev/plugins
 
 # Generated by tests
-sql-cli/tests/.airflow/dags/*
-!sql-cli/tests/.airflow/dags/.sql/.gitkeep
-!sql-cli/tests/.airflow/dags/.gitkeep
+sql-cli/tests/.airflow/.dags/*
```

Comment:
@feluelle, since we're adding this here and to pre-commit, could you already name things to what we agreed? I'm doing that in my PR, but it may be easier if we do as part of this one:
* `sql-cli/tests/.airflow/.dags/*` -> `sql-cli/tests/.airflow/dags/*`
* `sql-cli/tests/.airflow/.dags/.sql*` -> `sql-cli/tests/.airflow/dags/sql*`